Sources/LiveViewNative/Stylesheets/ParseableTypes/ContentTransition+ParseableModifierValue.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ContentTransition+ParseableModifierValue.swift
+//
+//
+//  Created by Carson Katri on 11/21/23.
+//
+
+import SwiftUI
+import Symbols
+import LiveViewNativeStylesheet
+
+extension ContentTransition: ParseableModifierValue {
+    public static func parser(in context: ParseableModifierContext) -> some Parser<Substring.UTF8View, Self> {
+//        identity, interpolate, numericText(countsDown:), numericText(value:), opacity, symbolEffect, symbolEffect(_:options:)
+        ImplicitStaticMember {
+            OneOf {
+                ConstantAtomLiteral("identity").map({ Self.identity })
+                ConstantAtomLiteral("interpolate").map({ Self.interpolate })
+                ConstantAtomLiteral("opacity").map({ Self.opacity })
+                ConstantAtomLiteral("opacity").map({ Self.opacity })
```

Comment:
I think this line was accidentally duplicated.

```suggestion
                ConstantAtomLiteral("opacity").map({ Self.opacity })
```